<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Userbar</class>
 <widget class="QWidget" name="Userbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="profile">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="lib_rs.qrc">:/image/profile</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="loginfo" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="usrIDLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="placeholderText">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pswLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginPushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>登 录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regPushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>注 册</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="adminRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>管理员</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="readerRadioButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>读者</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="usrinfo" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <widget class="QLabel" name="staLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="IDLabel">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="usrnameLabel">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>100</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="info_editPushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutPushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>注 销</string>
    </property>
   </widget>
  </widget>
  <zorder>loginfo</zorder>
  <zorder>usrinfo</zorder>
  <zorder>profile</zorder>
 </widget>
 <resources>
  <include location="lib_rs.qrc"/>
 </resources>
 <connections/>
</ui>
